### PR TITLE
Added tests

### PR DESCRIPTION
**Pull request by btnguyen2k**
- Tests are organized in their own package under directory `v2/module_test/`.
- Tests added in this PR: tests for integers, real numbers and money data types.

Start an Oracle instance locally with Docker:

```sh
$ docker run --restart unless-stopped -d --name oracle-xe -p 1521:1521 -e ORACLE_PWD=SecretPassword1 container-registry.oracle.com/database/express:21.3.0-xe
```

Run tests:

```sh
$ export ORACLE_DSN="oracle://system:SecretPassword1@localhost:1521/?SID=XE"
$ cd v2/module_test
$ go test -v -count 1 -p 1 ./
```

P/S: if a value stored in a floating point column (e.g. `NUMERIC(x,y)`) have zero fraction part (e.g. `123.00`) then it is fetched as an integer (e.g. Go's `int/uint`).
I consider this is a bug because the fetched value should be in Go's `float32/float64`. Hence some tests currently fails.

### CODE REVIEW

**Comment by btnguyen2k** [comment]
More tests (strings, unicode, date/time/timezone) are coming when I am available after New Year.

Merry Xmas & Happy New Year

File: v2/module_test/int_test.go [hunk 1]
```diff
@@ -0,0 +1,223 @@
+package v2_test
```
Comment (rondales) [inline v2/module_test/int_test.go:1]: v2/module_test/int_test.go

File: v2/module_test/go.mod [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module v2_test
```
Comment (rondales) [inline v2/module_test/go.mod:1]: .gitignore

**Comment by sijms** [comment]
Happy new year and thanks for your contributions
